worldengine/simulations/temperature.py

```diff
@@ -34,28 +35,61 @@ def _calculate(world, seed, elevation, mountain_level):
         base = rng.randint(0, 4096)
         temp = numpy.zeros((height, width), dtype=float)
 
-        from noise import snoise2
+        '''
+        Set up variables to take care of some orbital parameters:
+         distance_to_sun: -Earth-like planet = 1.0
+                          -valid range between ~0.7 and ~1.3
+                                see https://en.wikipedia.org/wiki/Circumstellar_habitable_zone
+                          -random value chosen via Gaussian distribution
+                                see https://en.wikipedia.org/wiki/Gaussian_function
+                          -width of distribution around 1.0 is determined by HWHM (half width at half maximum)
+                          -HWHM is used to calculate the second parameter passed to random.gauss():
+                                sigma = HWHM / sqrt(2*ln(2))
+                          -*only HWHM* should be considered a parameter here
+                          -most likely outcomes can be estimated:
+                                HWHM * sqrt(2*ln(10)) / sqrt(2*ln(2)) = HWHM * 1.822615728;
+                                e.g. for HWHM = 0.12: 0.78 < distance_to_sun < 1.22
+         axial_tilt:      -the world/planet may move around its star at an angle
+                                see https://en.wikipedia.org/wiki/Axial_tilt
+                          -a value of 0.5 here would refer to an angle of 90 degrees, Uranus-style
+                                see https://en.wikipedia.org/wiki/Uranus
+                          -this value should usually be in the range -0.15 < axial_tilt < 0.15 for a habitable planet
+        '''
+        distance_to_sun_hwhm = 0.12
+        axial_tilt_hwhm = 0.07
+
+        #derive parameters
+        distance_to_sun = rng.normal(loc=1.0, scale=distance_to_sun_hwhm / 1.177410023)
+        distance_to_sun = max(0.1, distance_to_sun)  # clamp value; no planets inside the star allowed
+        distance_to_sun *= distance_to_sun  # prepare for later usage; use inverse-square law
```

Comment:
inverse-square is certainly a bit too much, but I don't know what to choose instead. sqrt() might be appropriate, maybe even a logarithm.